pkg/operator/resources/cluster/crd.go

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
Should this be named datavolume.go in case we will have other crds in the future?

Reply:
eh, maybe hopefully other crds won't be so heinous